package.json

```diff
@@ -19,7 +19,11 @@
     }
   },
   "lint-staged": {
-    "*.{ts,tsx}": [
+    "!(*d).ts": [
+      "eslint --fix --max-warnings 0",
+      "git add"
+    ],
```

Comment:
What does this config mean @kelvin-muchiri ?

Reply:
@moshthepitt This will match all files except those ending in **.d.ts**. This is from the [doc](https://github.com/okonet/lint-staged#filtering-files)

Reply:
will it also match things such as .css .md etc?

Reply:
This is after checking the warning being thrown closely and noticed it involved `.d.ts` files. 
![Screenshot from 2020-10-07 10-38-03](https://user-images.githubusercontent.com/19818819/96694384-1812c100-1391-11eb-9817-0ac0e7025d7f.png)

Reply:
@moshthepitt I've actually seen that this issue is addressed [here](https://github.com/okonet/lint-staged#how-can-i-ignore-files-from-eslintignore-)

Reply:
@kelvin-muchiri could you elaborate?

Reply:
> will it also match things such as .css .md etc?

No